File: tutorials/springboot/SpringRestZioHttpServer/src/main/scala/net/officefloor/tutorial/ziohttpserver/Application.scala
package net.officefloor.tutorial.ziohttpserver

import tools.jackson.databind.JacksonModule
import tools.jackson.module.scala.DefaultScalaModule
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.context.annotation.Bean

// START SNIPPET: tutorial
@SpringBootApplication
class Application {

  @Bean
  def scalaJacksonModule(): JacksonModule = DefaultScalaModule
}

object Application extends App {
  SpringApplication.run(classOf[Application], args: _*)
}
// END SNIPPET: tutorial

File: tutorials/reference/SpringWebFluxApp/src/main/java/net/officefloor/tutorial/springfluxapp/Application.java
package net.officefloor.tutorial.springfluxapp;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.ComponentScan;

@SpringBootApplication
@ComponentScan(basePackageClasses = InjectDependency.class)
public class Application {

	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}

}